libnavui-speedlimit/src/main/java/com/mapbox/navigation/ui/speedlimit/view/MapboxSpeedLimitView.kt

```diff
@@ -0,0 +1,250 @@
+
+    private fun getSpeedLimitSpannable(
+        state: SpeedLimitState.UpdateSpeedLimit
+    ): SpannableStringBuilder {
+        val formattedSpeedLimit = state.getSpeedLimitFormatter().format(state)
+        val sizeSpanStartIndex = getSizeSpanStartIndex(state.getSignFormat(), formattedSpeedLimit)
+
+        return SpannableStringBuilder(formattedSpeedLimit).also {
+            it.setSpan(
+                StyleSpan(Typeface.BOLD),
+                0,
+                formattedSpeedLimit.length, SPAN_EXCLUSIVE_EXCLUSIVE
+            )
+            it.setSpan(
+                RelativeSizeSpan(1.9f),
+                sizeSpanStartIndex,
+                formattedSpeedLimit.length,
+                SPAN_EXCLUSIVE_EXCLUSIVE
+            )
+        }
+    }
+
+    internal fun getSizeSpanStartIndex(signFormat: SpeedLimitSign, formattedString: String): Int {
+        return when (signFormat) {
+            SpeedLimitSign.MUTCD -> 0
+            SpeedLimitSign.VIENNA -> formattedString.indexOf("\n") + 1
+        }
+    }
+
+    internal fun getFinalDrawable(signFormat: SpeedLimitSign): LayerDrawable {
+        val drawableShape = getDrawableShape(signFormat)
+        val backgroundDrawable = backgroundDrawable(drawableShape, speedLimitBackgroundColor)
+        val borderDrawable = borderDrawable(
+            drawableShape,
+            speedLimitMutcdBorderColor,
+            speedLimitViennaBorderColor
+        )
+        val outerBackgroundInset = getOuterBackgroundInset(signFormat)
+        val borderInset = getBorderInset(signFormat)
+        val innerBackgroundInset = getInnerBackgroundInset(signFormat)
+        return LayerDrawable(
+            arrayOf(
+                backgroundDrawable,
+                borderDrawable,
+                backgroundDrawable
+            )
+        ).also {
+            it.setLayerInset(
+                0,
+                outerBackgroundInset,
+                outerBackgroundInset,
+                outerBackgroundInset,
+                outerBackgroundInset
+            )
+            it.setLayerInset(
+                1,
+                borderInset,
+                borderInset,
+                borderInset,
+                borderInset
+            )
+            it.setLayerInset(
+                2,
+                innerBackgroundInset,
+                innerBackgroundInset,
+                innerBackgroundInset,
+                innerBackgroundInset
+            )
+        }
+    }
+
+    private fun getOuterBackgroundInset(signFormat: SpeedLimitSign): Int {
+        return when (signFormat) {
+            SpeedLimitSign.VIENNA -> VIENNA_MUTCD_OUTER_BACKGROUND_INSET
+            SpeedLimitSign.MUTCD -> VIENNA_MUTCD_OUTER_BACKGROUND_INSET
+        }
+    }
+
+    private fun getBorderInset(signFormat: SpeedLimitSign): Int {
+        return when (signFormat) {
+            SpeedLimitSign.VIENNA -> VIENNA_MUTCD_BORDER_INSET
+            SpeedLimitSign.MUTCD -> VIENNA_MUTCD_BORDER_INSET
+        }
```

Comment:
Since it's returning the same `INSET`, can we just do 

`return VIENNA_MUTCD_BORDER_INSET`